scene/gui/control.cpp

```diff
@@ -3369,6 +3377,7 @@ void Control::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_pivot_offset"), &Control::get_pivot_offset);
 	ClassDB::bind_method(D_METHOD("get_custom_minimum_size"), &Control::get_custom_minimum_size);
 	ClassDB::bind_method(D_METHOD("get_parent_area_size"), &Control::get_parent_area_size);
+	ClassDB::bind_method(D_METHOD("is_managed_by_container"), &Control::is_managed_by_container);
```

Comment:
This doesn't need to be bound. Is there any use for users?

Reply:
I thought it could be useful when writing custom controls, but I have to admit I can't think of any specific examples myself.

Reply:
If a method doesn't need to be exposed and is not subject of the proposal, it shouldn't be exposed.

Reply:
Aight, I'll take care of that when I can.